public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -78,6 +78,37 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
           let didNotAnswerCount  = this.survey.participantCount;
           q.selectedAnalysis = q.genderAnalysis;
           for (const a of q.countAnalysis.results) {
+            if (q.subQuestions) {
+              let matchedQuestionConcepts: QuestionConcept[] = [];
+              matchedQuestionConcepts = q.subQuestions.filter(
```

Comment:
may as well merge this with the previous line, no point in initializing with `[]` then immediately overwriting it